prime-router/src/test/kotlin/serializers/Hl7SerializerTests.kt

```diff
@@ -113,37 +115,68 @@ NTE|1|L|This is a final comment|RE"""
     fun `test write a message with Receiver`() {
         val inputStream = File("./src/test/unit_test_files/ak_test_file.csv").inputStream()
         val schema = "primedatainput/pdi-covid-19"
-
-        val hl7Config = mockkClass(Hl7Configuration::class).also {
```

Comment:
HL7Configuration is a data structure and really doesn't need to be mocked. This change removes a lot test code.